lms/routes.py

```diff
@@ -43,7 +43,7 @@ def includeme(config):
     )
     config.add_route(
         "blackboard_api.files.via_url",
-        "/api/blackboard/courses/{course_id}/files/{file_id}/via_url",
+        "/api/blackboard/courses/{course_id}/files/via_url",
```

Comment:
This is a bit unsatisfactory. Botu a course ID and a file ID are needed to get a Via URL for a Blackboard file. So the `/api/blackboard/courses/{course_id}/files/{file_id}/via_url` route was nice because both the course ID and file ID were in the path. But we're using `blackboard://...` URLs as the file IDs and it's a pain to put a URL in the path part of another URL. So I've moved it out into a query param. Another option would be to leave it in the path but encode it